Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Note: FROM command is inserted by Makefile
+# TODO: As of Docker-CE 17.05, we could use this better approach instead:
+#ARG base_layer
+#FROM $base_layer
+
+MAINTAINER Ben Barsdell <benbarsdell@gmail.com>
+
+ARG make_args
+
+# Build the library
+WORKDIR /bifrost
+COPY . .
+RUN make clean && \
+    make -j16 $make_args && \
+    make doc && \
```

Comment:
Removing the make doc sounds like the right idea.

Indeed the make clean is important to avoid strange errors when the user already built the library outside. I'll look into what's going on with version.py.